etc/cloud-build-push-charts.sh

```diff
@@ -22,7 +22,7 @@ dir=/tmp/charts
 rm -fr $dir
 mkdir -p $dir
 cd $dir
-charts="frconfig opendj amster openam openidm openig postgres-openidm cmp-platform"
+charts="frconfig opendj amster openam openidm openig postgres-openidm cmp-platform web"
```

Comment:
"web" is a pretty generic name - you might want to swap it out for something a little more specific. "forgeops-ui" occurs to me but it's just a thought. No big deal but if this lives "web" might not be the best name long term.

Otherwise, looks great and I'm looking forward to giving it a try.